src/modal/docs/readme.md

```diff
@@ -8,6 +8,7 @@ The `$modal` service has only one method: `open(options)` where available option
 * `scope` - a scope instance to be used for the modal's content (actually the `$modal` service is going to create a child scope of a provided scope). Defaults to `$rootScope`
 * `controller` - a controller for a modal instance - it can initialize scope used by modal. Accepts the "controller-as" syntax in the form 'SomeCtrl as myctrl'; can be injected with `$modalInstance`
 * `controllerAs` - an alternative to the controller-as syntax, matching the API of directive definitions. Requires the `controller` option to be provided as well
+* `bindToController` - when used with `controllerAs` & set to `true`, it will bind the controller properties onto the `$scope` directly
```

Comment:
As it says, `$scope`. If you see the implementation, it becomes instantly clear. This is similar to how `bindToController` works in Angular for directives with isolate scope.